src/services/components/guestBook.jsx

```diff
@@ -0,0 +1,73 @@
+//imports
+import React, { useState } from 'react';
+import { useEntries } from '../../context/EntryContext';
+import { useUser } from '../../context/UserContext';
+import './guestBook.css';
+
+export default function GuestBook() {
+  const [name, setName] = useState('');
+  const [messageEntry, setMessageEntry] = useState('');
+  const { user, setUser } = useUser();
+  const { message, setMessage } = useEntries();
+
+  function updateMessageList() {
+    if (!messageEntry) return;
+    setUser(name);
+    setMessage([...message, { name, message: messageEntry }]);
+    setMessageEntry('');
+  }
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    updateMessageList();
+  };
+
+  const userNameInput = (
+    <div>
```

Comment:
usually lonely divs with no classes are a bad sign. either use a fragment if you're only trying to group the return statement or use semantic html to describe the purpose like "section" or "li"